src/components/StartWithEthereumFlow/LetUseSomeApps/index.tsx

```diff
@@ -0,0 +1,168 @@
+import OpenSeaImage from "@/public/images/dapps/opensea.png"
+import UniswapImage from "@/public/images/dapps/uni.png"
+import WarpcastImage from "@/public/images/dapps/warpcast.png"
+
+const LetUseSomeApps = ({
+  stepIndex,
+  totalSteps,
+}: {
+  stepIndex: number
+  totalSteps: number
+}) => {
+  const dappsList = [
+    {
+      name: "Warpcast",
+      description: "The social and community platform of crypto.",
+      tag: (
+        <Tag
+          status="tag"
+          size="small"
+          className="bg-[#FFE3D3] font-bold text-black"
+        >
+          SOCIALS
+        </Tag>
+      ),
+      url: "https://warpcast.com/",
+      image: WarpcastImage,
+    },
+    {
+      name: "Aave",
+      description: "Lend your tokens to earn interest and withdraw any time.",
+      tag: (
+        <Tag
+          status="tag"
+          size="small"
+          className="bg-[#E1FEFA] font-bold text-black"
+        >
+          FINANCE
+        </Tag>
+      ),
+      url: "https://aave.com/",
+      image: AaveImage,
+    },
+    {
+      name: "Uniswap",
+      description: "Swap your tokens for different ones globally.",
+      tag: (
+        <Tag
+          status="tag"
+          size="small"
+          className="bg-[#E1FEFA] font-bold text-black"
+        >
+          FINANCE
+        </Tag>
+      ),
+      url: "https://app.uniswap.org/",
+      image: UniswapImage,
+    },
+    {
+      name: "OpenSea",
+      description: "Buy, sell, discover, and trade limited-edition goods.",
+      tag: (
+        <Tag
+          status="tag"
+          size="small"
+          className="bg-[#D1D1FF] font-bold text-black"
+        >
+          COLLECTIBLES
+        </Tag>
+      ),
+      url: "https://opensea.io/",
+      image: OpenSeaImage,
+    },
+  ]
+
+  return (
+    <div className="flex flex-col items-center gap-8 lg:flex-row lg:gap-24">
+      <div className="flex flex-1 flex-col gap-14">
+        <div className="flex flex-col gap-5">
+          <div>
+            <Tag status="tag">
+              {stepIndex} / {totalSteps}
+            </Tag>
+          </div>
+          <h2 className="text-3xl font-bold">Let Use Some Apps</h2>
+          <p>
+            Its time to go onchain and benefit from the wide ecosystem of
+            projects available you.
+          </p>
+          <div className="hidden lg:flex">
+            <Link href="/dapps" className="font-bold no-underline">
+              Explore more
+            </Link>
+          </div>
+        </div>
+      </div>
+      <div className="flex flex-1 flex-col gap-8">
+        <div>
+          {dappsList.map((dapp) => (
+            <div
+              key={dapp.name}
+              className="group flex cursor-pointer flex-col items-center justify-between gap-4 rounded-xl border-b border-background p-4 last:border-b-0 hover:bg-background-highlight sm:flex-row"
+              onClick={() => {
+                window.open(dapp.url, "_blank")
```

Comment:
Can't we reuse existing components? for example, this is very similar as the dapps list in /dapps
![image](https://github.com/user-attachments/assets/05bd45bc-968a-437d-b1ee-c40850b0e5a2)

Reply:
I think for now I will just use the linkBox component for this. Probably should either make this style of product list a standard too, or standardize through the site I agree.